M2/practica/index.html

```diff
@@ -1,105 +1,241 @@
+                                                $('<td>').text(rta[k]['logo']),
+                                                $('<td>').text(rta[k]['id']),
+                                                $('<td>').text(rta[k]['razs']),
+                                                $('<td>').text(rta[k]['pres'].nombre + ' ' + rta[k]['pres'].apellido),
+                                                $('<td>').text(rta[k]['patri'])
+                                        )
+                                )
+                            });
+                            finished();
+                        });
+
+                    },
+                    error: function () {
+                        $('div#mensaje-alerta').attr("class", "alert alert-danger").text("El servidor se encuentra ocupado. Por favor aguarde e intente nuevamente.");
+                        $('#boton-buscar').attr("disabled", false);
                     }
+
                 });
-            });
-        </script>
-	</head>
-	<body>
-        <div class="jumbotron text-center">
-            <div class="container">
-                <div class="row">
-                    <div class="col-md-12">
-                        <h1>Telapong S.A.</h1>
-                        <p>Listado de clientes oficiales</p>
-                    </div>
-                </div>
+
+
+                /*var tablaClientesBody = $('#tabla-clientes').find('tbody');
+                 if (tablaClientesBody.find('tr').length == 1) {
+                 tablaClientesBody.html('');
+                 }
+                 var cantidadExistente = tablaClientesBody.find('tr').length;
+                 var i = cantidadExistente;
+                 while (i < clientes.length && (i - cantidadExistente) < 10) {
+                 var cliente = clientes[i];
+                 tablaClientesBody.append(
+                 $('<tr>').append(
+                 $('<td>').addClass('text-center').append(
+                 $('<img>').attr('src', cliente.logo)
+                 ),
+                 $('<td>').text(cliente.id),
+                 $('<td>').text(cliente.razonSocial),
+                 $('<td>').text(cliente.direccion.calle + ' ' + cliente.direccion.altura + ', ' + cliente.direccion.ciudad),
+                 $('<td>').text(cliente.presidente.nombre + ' ' + cliente.presidente.apellido),
+                 $('<td>').addClass('text-right').addClass(cliente.activo >= cliente.pasivo ? 'success' : 'danger').text('$ ' + (cliente.activo - cliente.pasivo).toFixed(2)),
+                 $('<td>').text(cliente.fechaCreacion)
+                 )
+                 );
+                 i++;
+                 }
+                 if (tablaClientesBody.find('tr').length >= clientes.length) {
+                 $('#boton-buscar').attr('disabled', 'disabled');
+                 }*/
+
+            })
+            ;
+        })
+        ;
+    </script>
+</head>
+<body>
+<div class="jumbotron text-center">
+    <div class="container">
```

Comment:
Yo se que estas copiando este codigo de la practica de lea, pero lea flashea y no hay que hacerle caso a todo lo que dice. Ante duda de para que sirve el container, preguntale al verdadero maestro, google.
http://getbootstrap.com/css/ seccion Container:
"Note that, due to padding and more, neither container is nestable."
Traducido, el container viene inmediatamente despues de body, y toda la pagina va adentro del container. El container es unico en toda la pagina.